android/engine/src/test/resources/plans/disease-followup/patient.json

```diff
@@ -0,0 +1,16 @@
+{
+  "resourceType": "Patient",
+  "id": "Patient/TEST_PATIENT/_history/1",
+  "active": true,
+  "name": [
+    {
+      "family": "Hadi",
+      "given": [
+        "Bareera"
+      ]
+    }
+  ],
+  "gender": "female",
+  "birthDate": "2022-01-14",
+  "generalPractitioner":[{"reference":"Practitioner/TEST_PRACTITIONER"}]
+}
```

Comment:
Do we have a generic sampel patient that this test can use? That way we don't create a new patient resource for every test we write